.github/workflows/prepare-for-prod-deploy.yml

```diff
@@ -197,8 +197,12 @@ jobs:
           yq eval -i ".config-be-user-transformer.image.tag=\"$TAG_NAME\"" values.prod.yaml
           git add values.prod.yaml
 
+          cd ../../customer-objects/multi-tenant-us
+          yq eval -i ".transformer.image.tag=\"$VERSION\"" hostedmtedmt.yaml
+          yq eval -i ".user-transformer.image.tag=\"$VERSION\"" hostedmtedmt.yaml
```

Comment:
1. It seems like we are updating the image of transformer with this command. Is my understanding correct ?
2. Should we update the user-transformer also with the same version as destination-transformer ? Can you once clarify this one ?

Reply:
Also it seems like `UT_TAG_NAME` is being formed while creating pull request but I don't see it being used anywhere. Should we remove if it is not being used ?

Reply:
1. Yes
2. As of now, we are keeping the same tag for both transformer/ user-transformer for all clusters (enterprise, multi-tenant, pro). cc- @aashishmalik 
3. This is for future reference.

Reply:
Got it. Makes sense